single_tech_samples/datafactory/sample2_cicd_auto_publish/README.md

```diff
@@ -0,0 +1,322 @@
+# Automated Publish and CI/CD for Azure Data Factory
+
+This sample demonstrates the deployment of Azure Data Factory using Automated Publish Method. Usually, the Azure Data Factory deployment required Manual Publish setup where the developer had to manually Publish the ADF changes from the portal. This step generated the ARM Templates which were used in deployment step.
```

Comment:
Automated Publish Method => `Automated Publish` feature of CI/CD.

Reply:
required Manual Publish setup => requires `Manual Publish` setup

Reply:
developer had to manually Publish the ADF changes => developer publishes the ADF changes manually

Reply:
This step generated the ARM Templates which were used in deployment step. => This step generates ARM Templates which are used in deployment steps.